reana-ui/src/components/Notification.js

```diff
@@ -0,0 +1,57 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React, { useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Container, Message, Transition } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+import { clearError } from "../actions";
+import { getError } from "../selectors";
+
+import styles from "./Notification.module.scss";
+
+export default function Notification({ icon, message }) {
```

Comment:
Not to solve within this PR, but maybe to be ticketised if we agree:

Would it make sense to use the state for `message`(s) too? That way, the components that want to flash a message just push it to the state and `Notification` just shows it, as when an error occurs.

What do you think?

Reply:
Created a new issue to address it: https://github.com/reanahub/reana-ui/issues/123